README.md

```diff
@@ -1,15 +1,74 @@
 # Northcoders News API
 
-After cloning, follow these steps:
+Hosted version: https://nc-news-o4bo.onrender.com/
 
-1. create .env.test file and set PGDATABASE to the test database
+NC News or Northcoders News is a backend project to generate a functional API for a database of users, articles, comments, and topics for a fictional news website.
 
-    ```PGDATABASE=nc_news_test```
+If you wish to use this database yourself:
 
-2. create .env.dev file and set PGDATABASE to the dev database
+1. Clone the Repository
+    In the terminal on your device, cd into the directory where you wish to keep this repository, then,
 
-    ```PGDATABASE=nc_news```
+        ```git clone https://github.com/tjejack/be-nc-news.git```
 
-3. Add .env.* files to .gitignore
+    Or, if you have already forked this repo, copy the HTTPS link from the green 'Code' dropdown and replace the url.
 
-4. You're good to go! 
\ No newline at end of file
+2. Install the necessary dependencies
+    This repository uses npm, jest, dotenv, express, fs, and pg.
+    Before working with the repository, you're going to need to install these to your newly downloaded repo.
+
+    First, initialise npm by typing the following command into your terminal inside the git repository 
+
+        ```npm init -y```
+
+    Then, install jest and its additional matchers for the test suite. 
+
+    As dev dependencies, these jest files must be installed as such with the --save-dev command. See jest documentation  for more info: https://jestjs.io/docs/getting-started
+
+        ```npm install --save-dev jest```
+        ```npm install --save-dev jest-extended```
+        ```npm install --save-dev jest-sorted```
+
+    The dotenv package allows you to load environment variables from your .env files(see file setup). Read about dotenv here: https://www.npmjs.com/package/dotenv 
+
+        ```npm install dotenv --save```
+
+    Express is a framework which allows us to create our API more easily. See the documentation here: https://expressjs.com/
+
+        ```npm install express```
+
+    We will need both pg and pg-format in order to implement our SQL queries safely and reduce the risk of SQL injection.
+
+        ```npm install pg```
+        ```npm install pg-format```
+
+    Finally, we will need fs.promises package in order to asynchronously access our endpoints.json file at the /api endpoint.
+
+        ```npm install fs.promises```
```

Comment:
fs.promises comes with node, also they will only need to run `npm install` as that alone will install all the dependancies from the `package.json`